src/backend/common/models/user.py

```diff
@@ -122,6 +124,49 @@ def favorites_count(self) -> int:
             FavoriteQuery(account=none_throws(self._account), keys_only=True).fetch()
         )
 
+    """ Adds a Favorite for a model key/type. Returns Favorite if the model already exists """
+    def add_favorite(model_key: str, model_type: ModelType) -> Optional[Favorite]:
```

Comment:
- [ ] Confirm this code looks good as well and remove it from `mytba_helper.py` in py2